build.gradle.kts

```diff
@@ -19,9 +19,9 @@ hivemqExtension {
 }
 
 dependencies {
-    hivemqProvided("ch.qos.logback:logback-classic:${property("logback.version")}")
-    implementation("org.aeonbits.owner:owner:${property("owner.version")}")
-    implementation(platform("software.amazon.awssdk:bom:${property("aws-bom.version")}"))
+    hivemqProvided("ch.qos.logback:logback-classic:1.4.1")
```

Comment:
OpenSource projects will have then inline dependency versions? Should we document this somewhere? Has this being clarify with Silvio? 

FYI, for me inlining versions is perfectly fine.

Reply:
I was just experimenting, I converted the PR to draft.